bookwyrm/settings.py

```diff
@@ -37,6 +37,16 @@
 
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
+# Preview image
```

Comment:
Should I add a `ENABLE_PREVIEW_IMAGES` boolean switch? Some instance might want to spare the computing power, this feature is nice to have but not primordial to the app.

Reply:
I think that's a good idea as long as it isn't too much additional work